docker-compose.dev.yaml

```diff
@@ -1,5 +1,3 @@
-version: '3.8'
```

Comment:
Why did you remove it?

Reply:
When I ran the docker build command it recommended I remove it before I can run it. I'll add it back and fix other issues ASAP

Reply:
hmmm...
send me this error pleae, it's really weird